releasenotes/notes/0.19/py310-support-869d47583c976eef.yaml

```diff
@@ -5,7 +5,7 @@ features:
     precompiled binaries to PyPI for Python 3.10 on supported platforms.
 upgrade:
   - |
-    Qiskit Terra no longer fully supports 32 bit platforms on Python >= 3.10.
+    Starting from Python 3.10, Qiskit Terra will have reduced support for 32-bit platforms.
```

Comment:
In the rendered version this line read a little weirdly, because it appear right underneath the prelude message "We support Python 3.10", and it kind of (at first) sounded like we were dropping general support for 32-bit stuff.